pkg/controllers/netpol/pod.go

```diff
@@ -190,6 +190,17 @@ func (npc *NetworkPolicyController) setupPodNetpolRules(pod podInfo, podFwChainN
 		"-m", "addrtype", "--src-type", "LOCAL", "-d", pod.ip, "-j", "ACCEPT", "\n"}
 	npc.filterTableRules.WriteString(strings.Join(args, " "))
 
+	// ensure statefull firewall drops INVALID state traffic from/to the pod
+	// For full context see: https://bugzilla.netfilter.org/show_bug.cgi?id=693
+	// The NAT engine ingores any packet with state INVALID, because there's no reliable way to determine what kind of
```

Comment:
```suggestion
	// The NAT engine ignores any packet with state INVALID, because there's no reliable way to determine what kind of
```

Reply:
Fixed! Thanks for the spelling catch. I wonder why our misspell linter didn't catch that though.